app/src/partner/java/cm/aptoide/pt/store/view/home/HomeFragment.java

```diff
@@ -1,484 +0,0 @@
-        .loadWithCircleTransform(R.drawable.user_account_white, userAvatarImage);
-  }
-
-  private void setVisibleUserImageAndName(Account account) {
-    userEmail.setVisibility(View.VISIBLE);
-    userUsername.setVisibility(View.VISIBLE);
-    userEmail.setText(account.getEmail());
-    userUsername.setText(account.getNickname());
-    ImageLoader.with(getContext())
-        .loadWithCircleTransformAndPlaceHolder(account.getAvatar(), userAvatarImage,
-            R.drawable.user_account_white);
-  }
-
-  @Override public void onCreate(@Nullable Bundle savedInstanceState) {
-    super.onCreate(savedInstanceState);
-    getFragmentComponent(savedInstanceState).inject(this);
-    final AptoideApplication application =
-        (AptoideApplication) getContext().getApplicationContext();
-
-    searchNavigator =
-        new SearchNavigator(getFragmentNavigator(), application.getDefaultStoreName());
-
-    defaultThemeName = application.getDefaultThemeName();
-
-    drawerAnalytics = new DrawerAnalytics(analyticsManager, navigationTracker);
-    handleFirstInstall(savedInstanceState);
-
-    trendingManager = application.getTrendingManager();
-    crashReport = CrashReport.getInstance();
-    searchAnalytics = new SearchAnalytics(analyticsManager, navigationTracker);
-
-    setRegisterFragment(false);
-    setHasOptionsMenu(true);
-  }
-
-  @Override protected boolean hasSearchFromStoreFragment() {
-    return false;
-  }
-
-  @Override public void onDestroyView() {
-    userEmail = null;
-    userAvatarImage = null;
-    userUsername = null;
-    updatesBadge = null;
-    navigationView.setNavigationItemSelectedListener(null);
-    navigationView = null;
-    drawerLayout = null;
-    final Toolbar toolbar = getToolbar();
-    if (toolbar != null) {
-      toolbar.setNavigationOnClickListener(null);
-    }
-    unregisterClickHandler(backClickHandler);
-    super.onDestroyView();
-  }
-
-  @Override protected void setupViewPager() {
-    super.setupViewPager();
-
-    StorePagerAdapter adapter = (StorePagerAdapter) viewPager.getAdapter();
-
-    View layout = getTabLayout(adapter, Event.Name.myUpdates);
-    if (layout != null) {
-      updatesBadge = new BadgeView(getContext(), layout);
-    }
-
-    layout = getTabLayout(adapter, Event.Name.getUserTimeline);
-    if (layout != null) {
-      notificationsBadge = new BadgeView(getContext(), layout);
-    }
-
-    ((AptoideApplication) getContext().getApplicationContext()).getNotificationCenter()
-        .getUnreadNotifications()
-        .observeOn(Schedulers.computation())
-        .map(aptoideNotifications -> aptoideNotifications.size())
-        .distinctUntilChanged()
-        .observeOn(AndroidSchedulers.mainThread())
-        .subscribe(numberOfNotificationsUnread -> refreshBadge(numberOfNotificationsUnread,
-            notificationsBadge), throwable -> CrashReport.getInstance()
-            .log(throwable));
-
-    updateRepository.getNonExcludedUpdates()
```

Comment:
Method not being called anymore

Reply:
Will be used by Apps notification badge.